### PR TITLE
Issue/178 client transformations

### PR DESCRIPTION
**Pull request by jonase**

### CODE REVIEW

File: client/src/reducers/transforms/changeDatatype.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function makeParser(parser) {
   return (rows, idx, onError, args) => {
     const deleteMarker = {};
     const { defaultValue } = args;
-    const dv = defaultValue != null ? parser(defaultValue) : null;
+    const dv = defaultValue != null ? defaultValue : null;
```
Comment (gabemart) [inline client/src/reducers/transforms/changeDatatype.js]: Is the purpose of this to catch cases where defaultValue is undefined and convert it to null?

File: client/src/components/dataset/sidebars/TransformationLog.jsx [hunk 2]
```diff
@@ -10,9 +10,9 @@ function transformationDescription({ op, args }, columns) {
...
       return `${columnTitle(columnName, columns)} to lowercase`;
     case 'core/to-uppercase':
-      return `${columnTitle(columnName, columns)} to lowercase`;
+      return `${columnTitle(columnName, columns)} to uppercase`;
     case 'core/to-titlecase':
-      return `${columnTitle(columnName, columns)} to lowercase`;
+      return `${columnTitle(columnName, columns)} to titlecase`;
     case 'core/trim':
       return `${columnTitle(columnName, columns)} trimmed whitespace`;
     case 'core/trim-doublespace':
```
Comment (gabemart) [inline client/src/components/dataset/sidebars/TransformationLog.jsx]: Perhaps include a case for sorting. Something like

````    
case 'core/sort-column':
  return `${columnTitle(columnName, columns)} sorted ${args.sortDirection}`;
````

**Comment by gabemart** [comment]
LGTM aside from the small notes mentioned

File: client/src/reducers/transforms/changeDatatype.js (hunk 1 above)
Comment (jonase) [inline client/src/reducers/transforms/changeDatatype.js]: I'll change this to just let the default value flow through.. I missed this in the refactor. Good catch!

File: client/src/components/dataset/sidebars/TransformationLog.jsx (hunk 2 above)
Comment (jonase) [inline client/src/components/dataset/sidebars/TransformationLog.jsx]: Yeap I'll add this

**Comment by gabemart** [comment]
Looks good 👍

**Pull request merged**